charts/postgreslet/values.yaml

```
@@ -101,6 +101,14 @@ postgreslet:
   patroniRetryTimeout: 90
   # enableStandbyLeaderSelector enables the use of a new, updated selector for use with newer Spilo images. The postgreslet will use spilo-role=standby_leader in the selectors of standby DBs then.
   enableStandbyLeaderSelector: false
+  # enablePEtcd enables the deployment of a dedicated etcd for this postgreslet instance
+  enablePEtcd: true
+  # etcdImage The etcd image to use
+  etcdImage: "quay.io/coreos/etcd:v3.5.4"
+  # etcdBackupSidecarImage The sidecar image to use
+  etcdBackupSidecarImage: "ghcr.io/metal-stack/backup-restore-sidecar:latest"
```

I guess release version would be better: 

```suggestion
  etcdBackupSidecarImage: "ghcr.io/metal-stack/backup-restore-sidecar:v0.6.4"
```

---

I totally agree. This was the initial build of the chart, forgot to mark it as a draft :-)

But other than that, everything seems to be working as intended.

---

Very happy to see you working on this! 😍